jujugui/static/gui/src/app/components/env-switcher/test-env-switcher.js

```diff
@@ -184,10 +184,15 @@ describe('EnvSwitcher', function() {
       password: 'buffalo'
     }];
     var listEnvs = sinon.stub();
+    var listSrv = sinon.stub();
     var newEnv = sinon.stub();
+
+    listSrv.callsArgWith(0, [{path: 'admin/gui'}]);
```

Comment:
The value is completely arbitrary for this test--I can set something else if you like, but the real test is about the template and the state server matching.